components/sync15/src/request.rs

```diff
@@ -415,7 +444,8 @@ where
                 .into());
             }
             self.last_modified = last_modified;
-            self.batch = BatchState::Unsupported;
+            //Changed BatchState from Unsupported to NoBatch. Does this work?
```

Comment:
This is now a weird mismatch in state. I would say we should return the error directly.